aiosqlite/tests/smoke.py

```diff
@@ -354,6 +354,31 @@ async def test_iterdump(self):
                 ],
             )
 
+    async def test_cursor_on_closed_connection(self):
+        db = await aiosqlite.connect(TEST_DB)
+
+        cursor = await db.execute("select 1, 2")
+        await db.close()
+        with self.assertRaises(Exception):
+            await cursor.fetchall()
+        with self.assertRaises(Exception):
+            await cursor.fetchall()
```

Comment:
Would prefer to see something like `with self.assertRaisesRegex(ValueError, "connection closed")` to also ensure this catches the specific exception it's expecting to catch.